Update contributing guidelines for github template switch and github codespaces (#30)
dwave-examples/template: CONTRIBUTING.md

## Steps for Contributing

1. Follow GitHub's instructions to [create a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).

>    Alternatively, clone this repository (or one of our existing examples) for a basic
>    template from which to implement your example.

Reviewer: ```suggestion
   Alternatively, clone this repository (or one of our existing examples) as a 
   starting point from which to implement your example.
```

To prevent confusion with "template"

Once approved, we will make sure that your example is set up on our CircleCI
account.

To ensure your example runs in [GitHub Codespaces](https://docs.github.com/en/codespaces/overview),

> copy the `.devcontainer/` directory into your example's root directory.

Reviewer: ```suggestion
copy the [.devcontainer/](.devcontainer/) directory into your example's root directory.
```

I don't have time to test the above before my 1:00 meeting but I mean: can you make this a link instead?

Author: I was aiming for consistency with the line above, but it makes sense. I'll update the line above as well.
